lightning/src/ln/channelmanager.rs

```diff
@@ -483,7 +483,7 @@ impl<'a> Drop for PersistenceNotifierGuard<'a> {
 /// we, or our watchtower, must check for them having broadcast a theft transaction).
 pub(crate) const BREAKDOWN_TIMEOUT: u16 = 6 * 24;
 /// The amount of time we're willing to wait to claim money back to us
-pub(crate) const MAX_LOCAL_BREAKDOWN_TIMEOUT: u16 = 6 * 24 * 7;
+pub(crate) const MAX_LOCAL_BREAKDOWN_TIMEOUT: u16 = 2 * 6 * 24 * 7;
```

Comment:
Can you add the rational of why we're going to pick up 2016 ? "Max required value by some lightning implementation for wumbo channels (March 2021)"
That way 3 years from now we'll recall the why :)

Reply:
At risk of sounding pedantic, there should be a distinction:
- why a variable has a value belongs in the comment
- why and when a variable's value _changed_ belongs in the commit log

Reply:
I would say here we're interested about another software specific version. So our commit log should point to when/commit this other software was checked.